### PR TITLE
Make Cyborg and pAI able to use Vore

### PR DESCRIPTION
**Pull request by TearfulCircus**
## About The Pull Request
Give sillicons a verb for vore

## Why It's Good For The Game
The ash drake have it, dogborg have it, why wouldn't sillicons have 'em?

## Changelog
no

### CODE REVIEW

**Comment by ShadeAware** [comment]
░░░░░▒░░▄██▄░▒░░░░░░ 
░░░▄██████████▄▒▒░░░ 
░▒▄████████████▓▓▒░░ 
▓███▓▓█████▀▀████▒░░ 
▄███████▀▀▒░░░░▀█▒░░ 
████████▄░░░░░░░▀▄░░ 
▀██████▀░░▄▀▀▄░░▄█▒░ 
░█████▀░░░░▄▄░░▒▄▀░░ 
░█▒▒██░░░░▀▄█░░▒▄█░░ 
░█░▓▒█▄░░░░░░░░░▒▓░░ 
░▀▄░░▀▀░▒░░░░░▄▄░▒░░ 
░░█▒▒▒▒▒▒▒▒▒░░░░▒░░░ 
░░░▓▒▒▒▒▒░▒▒▄██▀░░░░ 
░░░░▓▒▒▒░▒▒░▓▀▀▒░░░░ 
░░░░░▓▓▒▒░▒░░▓▓░░░░░ 
░░░░░░░▒▒▒▒▒▒▒░░░░░░

**Comment by Strazyplus** [comment]
Pai is a tablet, how the fuck do they vore someone. its a hologram!

**Comment by TearfulCircus** [comment]
Hey, How can you vore someone? It is physically impossible to actually do that! 

Beside, It's hardlight. I'm not gonna explain that, thanks.

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
 	speech_span = SPAN_ROBOT
 	flags_1 = PREVENT_CONTENTS_EXPLOSION_1 | HEAR_1
-	vore_flags = NO_VORE
+	combat_flags = COMBAT_FLAGS_STAMEXEMPT_YESSPRINT /// Enable sprint system but not stamina
```
Comment (Akrilla) [inline code/modules/mob/living/silicon/silicon.dm:17]:
```suggestion
```

**Pull request merged**